AirCasting/SearchAndFollow/SearchMap/SearchMapViewModel.swift

```diff
@@ -46,14 +46,28 @@ class SearchMapViewModel: ObservableObject {
     func getMeasurementName() -> String { measurementType.capitalizedName }
     func getSensorName() -> String { sensorType.capitalizedName }
     
-    func locationNameInteracted(with newLocationName: String) {
+    func strokeColor(sessionID: Int) -> Color {
+        cardPointerID.number == sessionID ? Color.accentColor : .clear
+    }
+    
+    func enteredNewLocation(name newLocationName: String) {
         passedLocation = newLocationName
     }
     
-    func locationAddressInteracted(with newLocationAddress: CLLocationCoordinate2D) {
+    func enteredNewLocationAdress(_ newLocationAddress: CLLocationCoordinate2D) {
         passedLocationAddress = newLocationAddress
     }
     
+    func dismissedTapped() {
+        guard let currentPosition = currentPosition else {
+            Log.warning("Should never happen that current position is not present")
+            return
+        }
+        updateSessionList(geoSquare: currentPosition)
+        searchAgainButton = false
+        cardPointerID = .noValue
+    }
```

Comment:
internally there is no problem with just calling `redoTapped` inside this function

Reply:
Oh okay, I thought that it can be a bit different later that's why we should copy the implementation. Will change it :)